source/extensions/common/wasm/wasm.cc

```diff
@@ -1650,6 +1684,7 @@ void Wasm::registerCallbacks() {
     _REGISTER(___cxa_throw);
     _REGISTER(___cxa_pure_virtual);
     _REGISTER(___cxa_allocate_exception);
+    _REGISTER(___cxa_uncaught_exception);
```

Comment:
For my curiosity: could you explain why we need it here? Link is also good. Thanks!

Reply:
This is to support emscripten 1.38.39.  This will disappear from this PR when #83 is merged.